.travis.yml

```diff
@@ -68,6 +70,8 @@ matrix:
 script:
 - |
   set -e
+  # copy wrapper jar from cache, if it exists, but don't overwrite one in the source tree, if it exists:
+  false | cp -i $HOME/.m2/wrapper/maven-wrapper.jar $TRAVIS_BUILD_DIR/.mvn/wrapper/ 2>/dev/null || true
```

Comment:
Looking at this code now, I think this was a shell trick to ensure that the whole line was considered successful even if the `cp` failed. There's probably a better option.